README.md

```diff
@@ -138,6 +138,7 @@ Please see the [type specification](https://github.com/node-saml/node-saml/blob/
 - `additionalAuthorizeParams`: dictionary of additional query params to add to 'authorize' requests
 - `identifierFormat`: optional name identifier format to request from identity provider (default: `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`)
 - `wantAssertionsSigned`: if truthy, add `WantAssertionsSigned="true"` to the metadata, to specify that the IdP should always sign the assertions.
+- `wantAuthnResponseSigned` (default: true): if true, require that all incoming authentication response messages be signed at the top level, not just at the assertions. It is on by default. Note: either the response or the assertion must be signed even if both are turned off.
```

Comment:
This is good, but the note at then end should probably be on the other note for `wantAssertionsSigned` as well.

Reply:
Thanks @srd90.

@cjbarth i'm quite happy with @srd90 suggestion. How do you feel about doing that?

Reply:
I'd like to hear @markstos  thoughts, but we've been trying to remove properties from this README that are actually related to `node-saml` so that we don't have to maintain them in two places. I'd actually like to see continued effort in that direction with a note to indicate that the README at `node-saml` should be consulted.